weblab/templates/datasets/includes/dataset_header.html

```diff
@@ -11,14 +11,15 @@ <h1 id="entityname">
   <small>Created <time>{{dataset.created_at}}</time> by <em>{{dataset.author.full_name}}</em>.
     {% can_delete_entity dataset as can_delete %}
     {% if can_delete %}
-      Delete dataset:
+
       <a title="delete dataset" href="{% url 'datasets:delete' dataset.id %}">
+          Delete dataset:
         <img src="{% static 'img/delete.png' %}" alt="delete dataset"
              title="delete dataset"/></a>
     {% endif %}
 
     <a title="Transfer ownership of dataset" href="{% url 'datasets:transfer' dataset.id %}">
-      Transfer ownership </a>
+    | Transfer ownership |</a>
     <a title="Rename dataset" href="{% url 'datasets:rename' dataset.id %}">
         Rename dataset
       </a>
```

Comment:
Looking at the display of datasets compared with fitting specs, the latter has the linked protocol displayed at the entity header level (and called 'Linked protocol'), whereas datasets have it displayed inside the box (and called 'Corresponding protocol'). We should probably be consistent, and I think change the datasets to match fitting specs is best (since we shouldn't change where it sits for specs as it's a property of the whole spec not a single version).